frontend/e2e/auth.setup.ts

```diff
@@ -19,8 +19,7 @@ setup('auth setup', async ({ page }) => {
   await page.getByRole('textbox', { name: 'Password' }).fill(process.env.AUTH_PASSWORD)
   await page.getByRole('button', { name: 'Continue', exact: true }).click()
 
-  await page.getByRole('button', { name: 'Log In' }).click()
-  await page.getByRole('heading', { name: 'Home' }).waitFor()
+  await page.getByRole('button', { name: 'Log In' }).waitFor()
```

Comment:
For E2E tests, whilst playwright supports saving the state of indexedDB, it can only save things that are serialisable to JSON, and un-exportable keys are not, so I have to move the "Log in" click to every individual test

not sure of a nicer way to automate this, the manual solution is acceptable for now